### PR TITLE
#4 - Pricing request UI

### PR DESCRIPTION
**Pull request by ericfzeng**
Pricing menu:
- Pricing requests
      UI: Embed with iFrame wrapping Smartsheet form of "Sample of Pricing Request" 
      UI: Submit button: Integrate with Smartsheet API (Add Row) - TODO
- ISP Pricing requests
      UI: Integrate with Smartsheet API (List Rows) - TODO
- ISP Pricing responses
      UI: Integrate with Smartsheet API (Lost Rows) - TODO

### CODE REVIEW

**Comment by Dagolin** [comment]
@ericfzeng Github says it's conflicting, you have to `fetch` the latest commit and `rebase` your branch on the latest dev branch.

File: resources/views/layouts/partials/sidebar.blade.php [hunk 1]
```diff
@@ -53,13 +53,18 @@
...
-                    @endpermission
+                    @endpermission-->
                     @permission('isp')
                     <li><a href="#"><i class="fa fa-angle-double-right"></i>{{ trans('t4g.sidebar_isp_response') }}</a></li>
                     @endpermission
+
+                    <li><a href="{{ route('pricings.create') }}"><i class="fa fa-angle-double-right"></i>{{ trans('t4g.sidebar_customer_pricing') }}</a></li>
+                    <li><a href="{{ route('pricings.index') }}"><i class="fa fa-angle-double-right"></i>{{ trans('t4g.sidebar_isp_pricing') }}</a></li>
+                    <li><a href="{{ route('pricings.index') }}"><i class="fa fa-angle-double-right"></i>{{ trans('t4g.sidebar_isp_response') }}</a></li>
+
```
Comment (Dagolin) [inline resources/views/layouts/partials/sidebar.blade.php]: In my opinion, we need only one link to the `pricing request page`, and speaking of `create` link we can put it inside the `list` page just like what we did on `Customer / ISP pages`.

File: resources/views/pricings/isprequest.blade.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+@endsection
+
+@section('main-content')
+  @include('errors.error')
+  <div class="box box-primary">
+    {{ Form::open(['route' => 'pricings.store']) }}
+    @include('pricings.partials.form', ['submitButtonTitle' => 'Add Pricing'])
+    {{ Form::close() }}
+  </div>
+@endsection
```
Comment (Dagolin) [inline resources/views/pricings/isprequest.blade.php]: It looks like an unnecessary file?

Comment (ericfzeng) [inline resources/views/pricings/isprequest.blade.php]: Yes, this file is unnecessary and is deleted.

File: resources/views/layouts/partials/sidebar.blade.php (hunk 1 above)
Comment (ericfzeng) [inline resources/views/layouts/partials/sidebar.blade.php]: These 3 lines are removed. I will use the login of admin@telecom4good.org / admin as Paul suggested to show all the menu items.

**Comment by ericfzeng** [comment]
Hi Dago, conflicts resolved, please review.

File: resources/views/layouts/partials/sidebar.blade.php [hunk 3]
```diff
@@ -53,14 +53,14 @@
                 <ul class="treeview-menu">
                     @permission('customer')
                     <li><a href="#"><i class="fa fa-angle-double-right"></i>{{ trans('t4g.sidebar_customer_pricing') }}</a></li>
```
Comment (Dagolin) [inline resources/views/layouts/partials/sidebar.blade.php]: We need a hyperlink to the Pricing index page, try this `<a href="{{ route('pricings.index') }}">`, to see how many routes you have an their names, use  `php artisan route:list` in your terminal or command line tool (in PHPStorm Ctrl + Shift + X)

Comment (ericfzeng) [inline resources/views/layouts/partials/sidebar.blade.php]: Just pushed the changes: 
                <ul class="treeview-menu">
                    @permission('customer')
                    <li><a href="{{ route('pricings.create') }}"><i class="fa fa-angle-double-right"></i>{{ trans('t4g.sidebar_customer_pricing') }}</a></li>
                    @endpermission-->
                    @permission('isp')
                    <li><a href="{{ route('pricings.index') }}"><i class="fa fa-angle-double-right"></i>{{ trans('t4g.sidebar_isp_pricing') }}</a></li>
                    @endpermission-->
                    @permission('isp')
                    <li><a href="{{ route('pricings.index') }}"><i class="fa fa-angle-double-right"></i>{{ trans('t4g.sidebar_isp_response') }}</a></li>
                    @endpermission
                 </ul>

Comment (ericfzeng) [inline resources/views/layouts/partials/sidebar.blade.php]: Sorry, the copy/paste directly from PhPStorm doesn't show all the lines in GitHub. Here are all the lines:
                <ul class="treeview-menu">
                    @permission('customer')
                    <li><a href="{{ route('pricings.create') }}"><i class="fa fa-angle-double-right"></i>{{ trans('t4g.sidebar_customer_pricing') }}</a></li>
                    @endpermission-->
                    @permission('isp')
                    <li><a href="{{ route('pricings.index') }}"><i class="fa fa-angle-double-right"></i>{{ trans('t4g.sidebar_isp_pricing') }}</a></li>
                    @endpermission-->
                    @permission('isp')
                    <li><a href="{{ route('pricings.index') }}"><i class="fa fa-angle-double-right"></i>{{ trans('t4g.sidebar_isp_response') }}</a></li>
                    @endpermission
                 </ul>

Comment (ericfzeng) [inline resources/views/layouts/partials/sidebar.blade.php]: If this PR could be approved, should I go ahead to work on the "Pricing Requests" sub-menu item, which is, when user clicks Submit on the web form, save it to the Smartsheets as a new row.

**Pull request merged**